ui/components/app/confirm/info/row/row.tsx

```diff
@@ -40,6 +40,7 @@ export type ConfirmInfoRowProps = {
   copyEnabled?: boolean;
   copyText?: string;
   'data-testid'?: string;
+  collapsible?: boolean;
```

Comment:
Can we not re-use the existing `ui/components/app/confirm/info/row/expandable-row.tsx`?

Reply:
The designs are very different from expandable row.